akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/scaladsl/GrpcReadJournal.scala

```diff
@@ -310,7 +312,9 @@ final class GrpcReadJournal private (
       eventOffset.timestamp.toEpochMilli,
       eventMetadata = metadata,
       PersistenceId.extractEntityType(event.persistenceId),
-      event.slice)
+      event.slice,
+      filtered = false,
+      source = "")
```

Comment:
Source must be set to "BT" when it's a backtracking event. Use same constants in `EnvelopeOrgin`.

Reply:
I can't be cause we don't have a dependency on r2dbc except for from tests yet. I'll define a parallel constant for now.